docs_src/design/adr/0024-system-events.md

```diff
@@ -0,0 +1,93 @@
+# System Events ADR
+### Submitters
+- Lenny Goodell (Intel)
+
+## Change Log
+- [pending](https://github.com/edgexfoundry/edgex-docs/pull/795) (2022-07-12)
+
+## Referenced Use Case(s)
+- [System Events for Devices ](https://docs.edgexfoundry.org/2.3/design/ucr/0001-System-Events-for-Devices/)
+
+## Context
+System Events, aka Control Plane Events (CPE),  are new to EdgeX. This ADR addresses the [System Events for Devices](https://docs.edgexfoundry.org/2.3/design/ucr/0001-System-Events-for-Devices/) use case with an extensible design that can address other System Event use cases that may be identified in the future. This extensible design approach and the fact that System Events are produced and consumed by different EdgeX services makes it architecturally significant warranting this ADR.
+
+## Proposed Design
+To address the [System Events for Devices ](https://docs.edgexfoundry.org/2.3/design/ucr/0001-System-Events-for-Devices/) use case, Core Metadata will publish a new `SystemEvent` DTO to the EdgeX MessageBus when a device is added, updated or deleted. Consumers of these System Events will subscribe to the MessageBus to receive the new `SystemEvent` DTO .
+
+#### Data Transfer Object (DTO)
+
+This new `SystemEvent` DTO will contain the following data describing the System Event:
+
+- Event Type - i.e. device
+- Event Action - i.e. add/update/delete
+- Event Source - i.e core-metadata as the publisher
+- Event Owner - i.e device-onvif-camera as the device owner
+    - Optional based on the Event Type
+- Event Other- i.e. onvif-camera (profile name)
+    - Optional based on the Event Type
+- Event Details - i.e. Device DTO of newly added device or empty for Delete action.
+    - Optional and varies based on the Event Type and or Event Action. 
+
+#### MessageBus 
+
+Services that publish System Events (Core Metadata) must connect to the EdgeX MessageBus and have MessageBus configuration similar to that of Core Data's [here](https://github.com/edgexfoundry/edgex-go/blob/main/cmd/core-data/res/configuration.toml#L53-L74). This design assumes that Core Metadata will have this capability and configuration due to planned implementation of Service Metrics. 
+
+For this design, the `publish topic prefix` for System Events will be a constant in the code rather than another configuration item that never changes.
+
+#### MessageBus Topic
+
+The new `SystemEvent` DTO will be published to a multi-level topic allowing subscribers to filter by topic. The format of this topic for Device System Events will be:
+
+​		`edgex/system-event/{event-type}/{event-action}/{event-source}/{event-owner}/{event-other}`
```

Comment:
`Event Source` is what you are calling `Event Sender`. I agree that the order you have is better.

`edgex/system-event/{event-source}/{event-type}/{event-action}//{event-owner}/{event-other}`

so for Device System Events ONVIF Add example it would be:
`edgex/system-event/core-metadata/device/add/device-onvif-camera/onvif-camera`

Reply:
Reworked